yet-another-docker-java/pom.xml

```diff
@@ -61,11 +61,11 @@
             <artifactId>docker-java-core</artifactId>
             <version>${docker-java.version}</version>
         </dependency>
-        <dependency>
+<!--    <dependency>
             <groupId>com.github.docker-java</groupId>
             <artifactId>docker-java-transport-jersey</artifactId>
             <version>${docker-java.version}</version>
-        </dependency>
+        </dependency> -->
```

Comment:
what's wrong with jersey? This type of connector is working and has corresponding classes that will fail AFAIR

Reply:
With this dependency, my project is not being built locally:

`[WARNING] Rule 4: org.apache.maven.plugins.enforcer.RequireUpperBoundDeps failed with message:
Failed while enforcing RequireUpperBoundDeps. The error(s) are [
Require upper bound dependencies error for org.apache.httpcomponents:httpcore:4.4.10 paths to dependency are:
+-com.github.kostyasha.yet-another-docker:yet-another-docker-plugin:0.2.1-SNAPSHOT
  +-com.github.kostyasha.yet-another-docker:yet-another-docker-java:0.2.1-SNAPSHOT
    +-com.github.docker-java:docker-java-transport-jersey:3.2.0
      +-org.apache.httpcomponents:httpcore:4.4.10
and
+-com.github.kostyasha.yet-another-docker:yet-another-docker-plugin:0.2.1-SNAPSHOT
  +-com.github.kostyasha.yet-another-docker:yet-another-docker-java:0.2.1-SNAPSHOT
    +-com.github.docker-java:docker-java-transport-jersey:3.2.0
      +-org.apache.httpcomponents:httpclient:4.5.11
        +-org.apache.httpcomponents:httpcore:4.4.13
]`